libraries/AP_HAL_ChibiOS/hwdef/sw-spar-f407/hwdef.dat

```diff
@@ -91,7 +91,7 @@ define GPS_MAX_RECEIVERS 1
 define GPS_MAX_INSTANCES 1
 
 # rangefinder
-define HAL_PERIPH_ENABLE_RANGEFINDER 2
+define AP_PERIPH_RANGEFINDER_ENABLED 1 2
```

Comment:
This doesn't look right.  And does "2" do something?!

Reply:
Fixed that. I did check it though, but there are so many of them that this one slipped through my sight again. I'll be more careful next time.

Reply:
Fixed that. I did check it though but again there are so many of them that this one again escaped my sight. Will be more careful this time.